admin/publish-release.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
These are them ;-)

Good point--when they are rolled out elsewhere, add a comment that common contains the official versions?